<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCategoryDialog</class>
 <widget class="QDialog" name="EditCategoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>593</width>
     <height>262</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="uHorizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="uVerticalLayout">
      <item>
       <layout class="QHBoxLayout" name="uTitleHorizontalLayout">
        <item>
         <widget class="QLabel" name="uTitleLabel">
          <property name="text">
           <string> Title            </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="uTitleForm">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="uTagHorizontalLayout">
        <item>
         <widget class="QLabel" name="uTagLabel">
          <property name="text">
           <string> Tag             </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="uTagForm">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="uDescriptionHorizontalLayout">
        <item>
         <widget class="QLabel" name="uDescriptionLabel">
          <property name="text">
           <string> Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="uDescriptionForm"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="uCategoryListView"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>491</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="uApplyButton">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="uButtonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Close</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>uButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditCategoryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditCategoryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
